index.css

```diff
@@ -0,0 +1,37 @@
+#main{
+    background-color: #EEEEEE;
+    padding: 15px;
+}
+
+body{
+    text-align: center ;
+}
+
+#title{
+    text-align: center;
+}
+
+#img-div{   
+    padding: 10px;
+    background-color: white;
+}
+#image{
+    max-width: 100%;
+    height: auto;
+}
+
+#tribute-info{
+    justify-content: center;
+    text-align: center;
+    max-width: 500px;
+    margin: 0 auto 50px auto;
+}
+
+ul{
```

Comment:
Please try to use selectors instead of using tags while styling. Selectors help us specifically identify the tag to be styled to avoid styling other unrelated tags.

Reply:
- [Optionnal] Kindly use a couple of emojis and/or gifs in the review.
They make the code review more human-friendly 👽🤖.